BioArchLinux/spades/PKGBUILD

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
I don't think one will run `PKGBUILD` as a bash script, so the shebang is not needed here.

Reply:
Yes. I will remove it.